docs/CORTX-S3 Server Quick Start Guide.md

```diff
@@ -170,6 +170,10 @@ Before your test your build, ensure that you have installed and configured the f
     - If you don't have Python Version 3.3, then install python3 using: `$ yum install python3`
 4. Ensure that CORTX-S3 Server and its dependent services are running.
     1. To start CORTX-S3 Server and its dependent services, run the command: `$ ./jenkins-build.sh --skip_build --skip_tests`
+       
+       In case you recieve errors, follow the steps below:

```

Comment:
'errors related to haproxy not being running'

Reply:
Done, @nileshgovande, please check.